libs/langchain/langchain/vectorstores/weaviate.py

```diff
@@ -150,6 +150,8 @@ def add_texts(
                 data_properties = {self._text_key: text}
                 if metadatas is not None:
                     for key, val in metadatas[i].items():
+                        # Check if an id property is provided and rename it to 'textid'
+                        key = "textid" if key == "id" else key
```

Comment:
should we log a warning? also "textid" isn't super intuitive imo, maybe just "_id"?

Reply:
an exception might be better in case users skip reading warnings.

consider what what happens if later the user wants to do a filtered search using "id" and get's unexpected results.

Reply:
Hello,

If you try to push to Weaviate as it is, you would already receive an exception. This should be enough to prompt the person to take appropriate action. As for the key part, the idea was to avoid confusion with the automatically generated UUID of Weaviate, so i don't think using '_id' would be a good idea.

Kind regards,
Wanis

Reply:
I'm fine with whatever is decided here. Just thought it would be handy to help others.